### DISCUSSION TITLE
How to handle metadata of multi-country datasets

### DISCUSSION THREAD

**Discussion by gmartirano** [gopeg/issues-barriers#27] [General]
We should discuss aspects related to:
- Who is the metadata point of contact and with which role?
- Who is the dataset point of contact and with which role?
- Who will publish the metadata and where?
- How to ensure that GO-PEG output datasets will be finally discoverable in the EDP, e.g. through keywords?

Creating a GO-PEG project catalogue can be the solution?

**Comment by thorsten-reitz** [comment]
The way we intend to solve this for sustainability is to create a dataset series. In it, each child has metadata indicating the original organisation, whereas WE appears as the Distributor of the dataset series itself.

**Comment by AndersOstmanNovogit** [comment]
This issue is described in section 4.5.5 in D4.3

I have added the following text about potential solutions
Alternative solutions: The solutions here described are based on the assumption that the license conditions of the datasets allow for republishing and/or modification. 
One problem deals with the recognition of virtual entities (shortly described as entities having a web site but no bank account). A project like GO-PEG may be considered as a virtual entity. To be able to publish results in national catalogues, they have to be opened up to non-public organisations such as companies and preferably also to virtual entities. A survey among portal providers in Europe shows that this situation varies, where for instance Italy allows only public agencies to publish metadata in their catalogues while Sweden allows also for virtual entities. It should also be stated that there are discussions in Italy to change their policies in this respect.
As a summary, if license and other conditions allows and if the catalogue owner also allows entries from virtual entities, a project like GO-PEG may undertake the role of being the publisher of the value-added dataset. If these conditions can not be met, creating a project-specific catalogue may be one solution. The new catalogue standard OGC-API-for-records may facilitate this better.
In the case of mashups, one solution may be to create a dataset series. In it, each child has metadata indicating the original organisation, whereas the provider of the merged dataset appears as the distributor of the dataset series itself.

@gmartirano Which solution did you apply? Special agreement, or ...?

**Reply by AndersOstmanNovogit** [reply]
Now in D4.3 section 4.4.5
